utils/interfaces/_library/miscs.py

```diff
@@ -125,12 +125,14 @@ def check(self, data):
             self.log_error(f"{data['log_filename']} content should be an array")
             return
 
+        diagnostics = ["Diagnostics:"]
         span_types = []
         span_count = len(span_types)
 
         for span in _get_spans_by_rid(self.rid, data):
             span_count = span_count + 1
             span_types.append(span.get("type"))
+            diagnostics.append("{0}".format(span))
```

Comment:
f-string python rocks 😎 

```suggestion
            diagnostics.append(f"{span}")
```

Reply:
and even simpler : 
```suggestion
            diagnostics.append(str(span))
```